### PR TITLE
[Naitei] [Admin] Delete category

### PR DESCRIPTION
**Pull request by SuKay712**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/78664

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)

Screenshot from 2024-07-30 19-46-
![Screenshot from 2024-07-30 17-38-29
58](https://github.com/user-attachments/assets/478c74d3-5ea3-47e0-bbe4-2be79fe49405)

### CODE REVIEW

**Comment by SuKay712** [comment]
ready

File: app/views/admin/categories/_category.html.erb [hunk 1]
```diff
@@ -1,8 +1,17 @@
 <tr class="table-row">
+  <td class="table-data"><%= category_counter + 1 %></td>
```
Comment (ThaiKhacDuoc) [inline app/views/admin/categories/_category.html.erb]: Đưa vào Setting giúp bạn với nha
`1`

**Review by ThaiKhacDuoc — COMMENTED** [review]
cmt

File: app/views/admin/categories/_category.html.erb (hunk 1 above)
Comment (SuKay712) [reply app/views/admin/categories/_category.html.erb]: ok

**Review by ghost — COMMENTED** [review]
mergred

**Pull request merged**